webview/src/getStarted/components/NoData.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import pyTorch from '../snippets/pyTorch.py'
+import huggingFace from '../snippets/huggingFace.py'
+import keras from '../snippets/keras.py'
+import pythonApi from '../snippets/pythonApi.py'
+import { CodeBlock } from '../../shared/components/codeBlock/CodeBlock'
+import { CodeSlider } from '../../shared/components/codeSlider/CodeSlider'
+import { EmptyState } from '../../shared/components/emptyState/EmptyState'
+
+export const NoData: React.FC = () => {
+  return (
+    <EmptyState>
+      <div>
+        <h1>Your project contains no data</h1>
+        <div>
+          Enable DVC experiment tracking using{' '}
```

Comment:
Should we insert a sentence or two about running DVC experiments the usual way? Or would that make things look complicated 🤔

Reply:
I copied this off the blog post. I also think it might be confusing the steps together as this would come after. From there I think it'd be easier to show than write about it.